config/reports.yml.sample

```diff
@@ -80,34 +80,30 @@ uwsgi:
 
 reports:
 
-  # If running behind a proxy server and Galaxy is served from a
-  # subdirectory, enable the proxy-prefix filter and set the prefix in
-  # the [filter:proxy-prefix] section above.
-  #filter-with: proxy-prefix
-
-  # If proxy-prefix is enabled and you're running more than one Galaxy
-  # instance behind one hostname, you will want to set this to the same
-  # path as the prefix in the filter above.  This value becomes the
-  # "path" attribute set in the cookie so the cookies from each instance
-  # will not clobber each other.
+  # If Galaxy Reports is served at a URL prefix and you are running more
+  # than one Galaxy Reports instance behind one hostname, you will want
+  # to set this to the same path as the prefix in the mount uWSGI
+  # configuration above. This value becomes the "path" attribute set in
+  # the cookie so the cookies from each instance will not clobber each
+  # other.
   #cookie_path: null
 
   # Verbosity of console log messages.  Acceptable values can be found
   # here: https://docs.python.org/2/library/logging.html#logging-levels
   #log_level: DEBUG
 
-  # Database connection Galaxy reports are intended for production
+  # Database connection. Galaxy Reports are intended for production
   # Galaxy instances, so sqlite (and the default value below) is not
   # supported. An SQLAlchemy connection string should be used specify an
   # external database.
   #database_connection: 'sqlite:///./database/universe.sqlite?isolation_level=IMMEDIATE'
 
-  # Where dataset files are saved Temporary storage for additional
-  # datasets, this should be shared through the cluster
+  # Where dataset files are saved. Temporary storage for additional
+  # datasets, this should be shared through the cluster.
   #file_path: database/files
 
-  # Where dataset files are saved Temporary storage for additional
-  # datasets, this should be shared through the cluster
+  # Where dataset files are saved. Temporary storage for additional
+  # datasets, this should be shared through the cluster.
   #new_file_path: database/tmp
```

Comment:
Not the fault of this PR, but the comments for `file_path` and `new_file_path` are duplicated and not correct for either. I'll push a correction.